A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/FCU/A320_Neo_FCU.js

```diff
@@ -34,13 +34,6 @@ class A320_Neo_FCU extends BaseAirliners {
         super.onFlightStart();
         if (this.mainPage)
             this.mainPage.onFlightStart();
-
-        // A workaround to allow us to default the FD to on when spawning on runway
-        if (SimVar.GetSimVarValue("L:A32NX_AUTOPILOT_FLIGHT_DIRECTOR_DEFAULT", "Bool") === 1) {
```

Comment:
please remove all instance of this var as well: L:A32NX_AUTOPILOT_FLIGHT_DIRECTOR_DEFAULT

Reply:
Done!